### ISSUE TITLE
convert tool shipped with ASP is buggy

### ISSUE THREAD

**Issue by oleg-alexandrov** [NeoGeographyToolkit/StereoPipeline#384]
The convert tool shipped with ASP can't do even something as simple as:

convert input.jpg output.jpg

To be looked into, when there is time. For now, it is suggested to use conda to fetch a functioning version of it, in a new environment, with the command:

conda install -c conda-forge imagemagick

and set the path to that one before invoking historical_helper.py or some other tool which needs it. If historical_helper.py refuses to use the new path and uses the old one, some lines on top of that script which set the path should be commented out, and then it will hopefully invoke the right tool.

**Comment by iamtekson** [comment]
Hi, I encountered similar issue. What is the best way to solve this issue using the latest version of `ASP` in linux machine? Please help me!

![image](https://github.com/NeoGeographyToolkit/StereoPipeline/assets/39838116/bf68655d-7fba-44ed-a506-1be5314c805f)

**Comment by iamtekson** [comment]
To be more specific, I am trying to process the `KH-9` imagery. I tried to run `historical_helper.py` command as below. But the library always generate the following error,

![error message KH-9 processing using ASP](https://github.com/NeoGeographyToolkit/StereoPipeline/assets/39838116/7d2612d8-645a-434a-97cc-54ec3aa7e0a0)

**Comment by oleg-alexandrov** [comment]
The just-released ASP 3.3.0 software no longer ships the convert tool with the zipped binaries. I just added documentation for how to install it, if needed.

The conda ASP package however has a 'convert' tool, but that is the wrong one, it is a helper program for a third party-library named "embree". That we cannot control.

So, in either case, it is best the user installs the convert tool separately, in a different environment. That is now documented here: https://stereopipeline.readthedocs.io/en/latest/tools/historical_helper.html

I also modified the historical_helper.py script to check for this tool, and print a message if the tool does not exist or the wrong tool is found, and suggesting installing it with conda.

These changes are after the release, they are available with the latest build at https://github.com/NeoGeographyToolkit/StereoPipeline/releases
